koku/api/settings/tags/mapping/view.py

```diff
@@ -35,6 +34,15 @@
 from reporting.provider.all.models import TagMapping
 
 
+def parse_list(string_to_parse):
```

Comment:
Wrote as a separate function for now because I will use it in the multi filter PR I have open. This is just a quick fix to close out COST-4786.